.github/workflows/license-file-gen.yaml

```diff
@@ -0,0 +1,82 @@
+name: License File Generation
+
+on:
+  workflow_call:
+    inputs:
+      repo:
+        description: 'Repository name'
+        required: true
+        default: 'aiverify-foundation/moonshot'
+        type: string
+      branch:
+        description: 'Branch or tag'
+        required: true
+        default: 'main'
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'Repository name'
+        required: true
+        default: 'aiverify-foundation/moonshot'
+        type: string
+      branch:
+        description: 'Branch or tag'
+        required: true
+        default: 'main'
+        type: string
+
+jobs:
+  license-file-gen:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.inputs.repo }}
+          ref: ${{ github.event.inputs.branch }}
+          submodules: recursive
+
+      - name: Setup Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+
+      - name: Generate license file
+        run: |
+          set +e
+          pip install -r requirements.txt
+          npm install
+          repo_name=$(echo "${{ github.event.inputs.repo }}" | awk -F'/' '{print $2}')
+          bash <(curl -s -L https://detect.synopsys.com/detect9.sh) \
+            --blackduck.url=https://blackduck.aipo-imda.net/ \
+            --blackduck.api.token=${{ secrets.BLACKDUCK_TOKEN }} \
+            --detect.detector.search.depth=10 \
+            --detect.project.name=aiverify-${repo_name} \
+            --detect.project.version.name=${{ github.event.inputs.branch }} \
+            --detect.source.path=. \
+            --detect.code.location.name="aiverify-${repo_name}_${{ github.event.inputs.branch }}" \
+            --detect.excluded.directories=/tests \
```

Comment:
Need to change to "--detect.excluded.directories=venv,tests"

Reply:
Hi, not sure why the need to exclude venv, I thot Blackduck should be scanning the actual dependencies installed?

Reply:
ah I see....if thats the case, just need to change to tests. There was a typo on the exclusion on the regex for matching the folders name.